redash/worker.py

```diff
@@ -93,3 +97,68 @@ def add_periodic_tasks(sender, **kwargs):
     for params in extensions.periodic_tasks.values():
         # Add it to Celery's periodic task registry, too.
         sender.add_periodic_task(**params)
+
+
+class HardTimeLimitingWorker(Worker):
```

Comment:
How about we move this into its own module in the `redash.tasks` package? The `redash.schedule` module belongs there as well. 

I would move all the rq stuff from `redash.worker` into `redash.tasks` and eventually remove this module when we say goodbye to Celery.

Reply:
Also worth adding some documentation on why we added this class.